scripts/pretty-printers/gdb/pretty_printers.py

```diff
@@ -125,8 +141,14 @@ def children(self):
                     if node.dereference()["_M_right"] != parent:
                         node = parent
 
+        # If we have no children it means we're actually a value node, so
+        # return the value of the node and a throw-away second value.
+        if sub_count == 0 and named_sub_count == 0:
+            yield (self.to_string(), self.val)
```

Comment:
Why not just not return any children?

Reply:
It needed this originally, doesn't anymore so has been removed.